TTS/vocoder/datasets/preprocess.py

```diff
@@ -1,17 +1,38 @@
 import glob
 import os
 from pathlib import Path
+from tqdm import tqdm
 
 import numpy as np
 
 
+def preprocess_wav_files(out_path, config, ap):
```

Comment:
where do you use this? I couldn't figure.

Reply:
It's used here [Line#327](https://github.com/mozilla/TTS/pull/542/commits/995d84f6d74fa3592fcb7cd5b31f9246155191a8#diff-499195b4c5f58700d3072e2be8ec03380c7399ee562a43c36919307bba1d105bR327) to preprocess the features if no path is set in the config and no features where found in the training folder. But as you stated bellow it is better to compute the data on the fly, I just wasn't sure about the overhead.